.github/workflows/bi-connectors.yml

```diff
@@ -0,0 +1,63 @@
+name: Build connectors for BI tools
+
+on:
+  push:
+    paths:
+      - 'bi-connectors/PowerBIConnector/**'
+      - 'bi-connectors/TableauConnector/**'
+      - '.github/workflows/bi-connectors.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Pack Tableau ODBC connector
```

Comment:
Why do we pack both ODBC and JDBC? I thought we were only going to use JDBC going forward?

Reply:
This script build both for future reference. At the moment, repo already contains a taco for ODBC, so nothing changed for ODBC actually.

Reply:
The ODBC one is now deprecated for the JDBC one, we should stop building the ODBC one.

Reply:
Disabled in 4cadfd18.